results/i9-12900H/v.semykin/g++-14/run_benchmarks.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+declare -A benchmarks=(
+    ["coro-fibs"]="coro-fibs/coro-fibs"
+    ["exc_ret"]="excret/exc_ret"
+    ["exc_ret_noexc"]="excret/exc_ret_noexc"
+    ["extswap"]="inline/extswap"
+    ["exc_partition"]="noexcept-qsort/exc_partition"
+    ["exc_qsort"]="noexcept-qsort/exc_qsort"
+    ["filter"]="ranges-filter/filter"
+    ["projector"]="ranges-projector/projector"
+    ["virtinh"]="virtual-inherit/virtinh"
+    ["virtinl"]="virtual-inline/virtinl"
+    ["virtual"]="virtual-overhead/virtual"
+    ["virtual-shuffle"]="virtual-overhead/virtual-shuffle"
+)
+
+base_dir="../../../../build/Release"
```

Comment:
What happens if I run it from directory other than the one this scripts exists in?

Reply:
> What happens if I run it from directory other than the one this scripts exists in?

Existence error of the executable. Therefore, u need to correct paths within script. It just a sample to help users